controllers/configurationpolicy_controller.go

```diff
@@ -1172,6 +1186,10 @@ func (r *ConfigurationPolicyReconciler) handleObjectTemplates(plc *policyv1.Conf
 		// map raw object to a resource, generate a violation if resource cannot be found
 		if decodeErrResult == nil {
 			scopedGVR, mappingErrResult = r.getMapping(desiredObj.GroupVersionKind(), plc, indx)
+
+			if mappingErrResult != nil {
+				mappingError = ErrMappingError
+			}
```

Comment:
It appears the first error case returned from `getMapping()` is a missing API version, which shouldn't be re-queued since it's an invalid Policy, so it'd be best to handle that one differently.

Reply:
Good catch! Fixed in the latest push.